xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -432,7 +432,11 @@ COverlay* CRenderer::Convert(CDVDOverlay* o, double pts)
 #endif
 
   if(!r && o->IsOverlayType(DVDOVERLAY_TYPE_TEXT))
-    r = new COverlayText(static_cast<CDVDOverlayText*>(o));
+  {
+    int targetWidth = MathUtils::round_int(m_rv.Width());
```

Comment:
no need to round, there are no fractions of pixels

Reply:
I just used the definition above for the libass subtitles (https://github.com/xbmc/xbmc/blob/fa2f9e70c1c03aa216daed641313336e37f98dd2/xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.cpp#L340). Should be removed there too?

Reply:
up to you if you want to invest the time.
MathUtils::round_int is a rather complicated implementation of something trivial. As far as I know this was originally created for GUI, then spread out like cancer because it was placed into generic utils. If you want to round a value, better use standard c++ functions.

Here we deal with a different issue. xy coordinates, which are integers, are put into a struct, CRect, that has float members. This is all a mess :)
A big problem is that some parts of the app rely on incorrect code of other parts. Means even if you do something right here other things may break.